tests/collections/llm/gpt/data/test_hf_mockdataset.py

```diff
@@ -13,12 +13,12 @@
 # limitations under the License.
 
 
+from unittest.mock import MagicMock, patch
```

Comment:
CodeQL says `patch` is unused?

Reply:
seems to be some merge conflct resolve issue. Let me revert. Thanks for checking this